src/bauhaus/bauhaus.h

```diff
@@ -275,6 +275,13 @@ void dt_bauhaus_widget_set_quad_visibility(GtkWidget *w,
 // set a tooltip for the quad button:
 void dt_bauhaus_widget_set_quad_tooltip(GtkWidget *w,
                                         const gchar *text);
+// helper macro to set all quad properties at once
+#define dt_bauhaus_widget_set_quad(w, paint, toggle, callback, tooltip)      \
```

Comment:
I was wondering whether we have a coding style guideline which states how macros should be named.
I'm asking because this file contains other macros, which is written in capital letter screaming snake case.
Can this be an inline function too?